src/main/resources/application.properties.EXAMPLE

```diff
@@ -25,6 +27,3 @@ metadb.publishing_failures_filepath=
 
 # detailed consistency checking logger filepath
 cmo_label_generator.request_logger_filepath=
-
-# cmo request filter (default is false)
-igo.cmo_request_filter=
```

Comment:
dont need this anymore since cmo filtering is happening upstream by request/sample validator